doc/companies.md

```diff
@@ -1 +1,980 @@
+    "name": "Test Company Center"
+}
+```
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id":"5",
+        "company_code": "TCC",
+        "name": "Test Company Center",
+        "address": "JL. Test Company",
+        "npwp": "123",
+        "nomor_kukuh": "123",
+        "tgl_kukuh": "12",
+        "phone_num": "021",
+        "fax_num": "021",
+        "seri_pajak": "12",
+        "kota": "JAKARTA"
+    }
+}
+```
+
+#### Case : Positive Case 3
+
+Request Payload :
+```
+{
+    "id": "5",
+    "address": "JL. Test Company Center"
+}
+```
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id":"5",
+        "company_code": "TCC",
+        "name": "Test Company Center",
+        "address": "JL. Test Company Center",
+        "npwp": "123",
+        "nomor_kukuh": "123",
+        "tgl_kukuh": "12",
+        "phone_num": "021",
+        "fax_num": "021",
+        "seri_pajak": "12",
+        "kota": "JAKARTA"
+    }
+}
+```
+
+#### Case : Positive Case 4
+
+Request Payload :
+```
+{
+    "id": "5",
+    "npwp": "1234"
+}
+```
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id":"5",
+        "company_code": "TCC",
+        "name": "Test Company Center",
+        "address": "JL. Test Company Center",
+        "npwp": "1234",
+        "nomor_kukuh": "123",
+        "tgl_kukuh": "12",
+        "phone_num": "021",
+        "fax_num": "021",
+        "seri_pajak": "12",
+        "kota": "JAKARTA"
+    }
+}
+```
+
+#### Case : Positive Case 5
+
+Request Payload :
+```
+{
+    "id": "5",
+    "phone_num": "0216"
+}
+```
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id":"5",
+        "company_code": "TCC",
+        "name": "Test Company Center",
+        "address": "JL. Test Company Center",
+        "npwp": "1234",
+        "nomor_kukuh": "123",
+        "tgl_kukuh": "12",
+        "phone_num": "0216",
+        "fax_num": "021",
+        "seri_pajak": "12",
+        "kota": "JAKARTA"
+    }
+}
+```
+
+## <a name="delete"></a>Delete Data Companies
+
+### Endpoint
+DELETE /companies/:id
+
+### Headers
+Key | Value
+--- | ---
+Content-Type | application/json
+Accept | application/json
+
+### Request Payloads
+Name | Type | Example Value
+--- | --- | ---
+id | string | 5
+```
+{
+    "id": "5",
+}
+```
+
+### Response Payloads
+HTTP Code | Status | Description
+--- | --- | ---
+400 | Bad Request | Bad request payload  
+404 | Not Found | User not found in database  
+500 | Internal Server Error | some un-handle error in server
+200 | OK | OK
+```
+{
+    "status_code": "CDC-400",
+    "status_message": "Bad Request",
+    "data": null
+}
+```
+
+```
+{
```

Comment:
no content in body

Reply:
remove all response payload. 204 no content it;s mean really no content ion body response.